content/departments/product-engineering/engineering/policies/vulnerability-management-policy.md

```diff
@@ -0,0 +1,142 @@
+- Triaging the vulnerability reports and confirming their impact and exploitability in our systems
+- Assigning a severity to confirmed vulnerabilities
+- Engaging with code owners to patch vulnerabilities
+- Verifying that vulnerabilities are properly fixed
+- Disclosing vulnerabilities
+- Adhering to the SLAs
+
+Code or system owners are responsible for:
+
+- Fixing vulnerabilities in code or system owned by them
+- Suggesting patches for vulnerabilities and estimating effort to do it
+- Patching vulnerabilities
+- Adhering to the SLAs
+
+The Security team will cooperate with and support all teams throughout the entire process.
+
+### Severity levels
+
+The severity assessment will be done with the full context of the impact to the business and not just the severity level in isolation, as an example a vulnerability detected in a Go library and marked as critical by a scanner does not automatically mean the severity is critical for the business. The main factors to assess are: **Exploitability, Asset Value, Likelihood and Business impact**.
+
+We classify our vulnerabilities in 4 severity levels:
+
+- **Critical**: vulnerabilities that result in a complete compromise of the infrastructure or application, unavailability of the application, or imminent compromise of highly sensitive information. Critical vulnerabilities can be exploited without elevated access or user interaction. Critical vulnerabilities can be exploited with relative ease, or low technical ability, or have the potential to be fully automated once a viable exploit has been created. Vulnerabilities that become a threat to the continuity or reputation of the business like data breach.
+- **High**: vulnerabilities that can have a significant impact on the operational integrity of the application, data security, authorization boundaries but require either a complex attack chain or user interaction for exploitation. High severity vulnerabilities usually have mitigating controls in place, such as requiring an admin account on the system or application to perform successful exploitation.
+- **Medium**: vulnerabilities that can impact the operational integrity of the application, limited unavailability or serve as a step in a complex attack chain. Medium severity vulnerabilities do not lead to high impact incidents single-handedly but can serve as an important step in complex attacks.
+- **Low**: vulnerabilities that leak information not considered highly sensitive, or issues that cannot be chained into attacks with high impact.
+
+### 1. Discovery
+
+As our security program grows we will continue to add tooling and processes that find vulnerabilities across our systems and code. Some sources of vulnerability information that we already have or are in our immediate plans are:
+
+- Penetration Testing
+- Bug Bounty Program
+- Vulnerability Scanning
+- CI/CD pipeline security checks
```

Comment:
Would it make sense to maintain a collection of code monitors, as well?

Reply:
Sure! I have one running for new usages of AuthzQueryConds but we will definitely expand this (and share 😄)